java/project/src/main/java/com/esri/arcgisruntime/sample/project/MainActivity.java

```diff
@@ -84,7 +84,7 @@ public boolean onSingleTapConfirmed(MotionEvent motionEvent) {
         Point originalPoint = mMapView.screenToLocation(clickedLocation);
         inputPointGraphic.setGeometry(originalPoint);
         // project the web mercator point to WGS84 (WKID 4236)
```

Comment:
let's remove the comment reference to 4236 since we're using the getWgs84 method